### PR TITLE
Wrldbaems 78 breadcrumb UI

### PR DESCRIPTION
**Pull request by akki761**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix # [WRLDBAEMS-78](https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-78)
Test URLs:
- Before: [https://main--world-bank--aemsites.hlx.live/en/drafts/akash/breadcrumb](https://main--world-bank--aemsites.hlx.live/en/drafts/akash/breadcrumb)
- After: [https://wrldbaems-78-breadcrumb-ui--world-bank--aemsites.hlx.live/en/drafts/akash/breadcrumb](https://wrldbaems-78-breadcrumb-ui--world-bank--aemsites.hlx.live/en/drafts/akash/breadcrumb)

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -156,6 +156,29 @@ export async function fetchLanguagePlaceholders() {
...
 }
 
+/**
+ * Helper function to create DOM elements
+ * @param {string} tag DOM element to be created
+ * @param {Object} attributes attributes to be added
+ * @param {HTMLElement|SVGElement|string} html HTML or SVG to append to/after new element
+ */
+export function createTag(tag, attributes, html = undefined) {
+  const el = document.createElement(tag);
```
Comment (mkbansal1) [inline scripts/scripts.js:166]: We have dom-helper.js to created tags. Dont create new method.

File: scripts/scripts.js [hunk 2]
```diff
@@ -278,6 +301,22 @@ function loadDelayed() {
...
 }
 
+/**
+ * Fetch filtered search results
+ * @returns List of search results
+ */
+export async function fetchSearch() {
+  window.searchData = window.searchData || {};
+  if (Object.keys(window.searchData).length === 0) {
+    const lang = document.documentElement.lang;
```
Comment (mkbansal1) [inline scripts/scripts.js:311]: Please use getLanguage() method of utils.js

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@akki761 Please fix lint issues

File: scripts/scripts.js (hunk 1 above)
Comment (akki761) [reply scripts/scripts.js:166]: ok..didn't know that. Will do

File: scripts/scripts.js (hunk 2 above)
Comment (akki761) [reply scripts/scripts.js:311]: sure

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**